tenable/io/v3/vm/exclusions/api.py

```diff
@@ -0,0 +1,443 @@
+            weekdays (list, optional):
+                List of 2-character representations of the days of the week to
+                repeat the frequency rule on.  Valid values are:
+                *SU, MO, TU, WE, TH, FR, SA*
+                Default values: ``['SU', 'MO', 'TU', 'WE', 'TH', 'FR', 'SA']``
+            day_of_month (int, optional):
+                The day of the month to repeat a **MONTHLY** frequency rule on.
+                The default is today.
+            enabled (bool, optional):
+                If enabled is true, the exclusion schedule is active.
+                If enabled is false, the exclusion is "Always Active"
+                    The default value is ``True``
+            network_id (uuid, optional):
+                The ID of the network object associated with scanners
+                where Tenable.io applies the exclusion.
+
+        Returns:
+            :obj:`dict`:
+                Dictionary of the newly minted exclusion.
+
+        Examples:
+            Creating a one-time exclusion:
+
+            >>> from datetime import datetime, timedelta
+            >>> exclusion = tio.v3.vm.exclusions.create(
+            ...     'Example One-Time Exclusion',
+            ...     ['127.0.0.1'],
+            ...     start_time=datetime.utcnow(),
+            ...     end_time=datetime.utcnow() + timedelta(hours=1))
+
+            Creating a daily exclusion:
+
+            >>> exclusion = tio.v3.vm.exclusions.create(
+            ...     'Example Daily Exclusion',
+            ...     ['127.0.0.1'],
+            ...     frequency='daily',
+            ...     start_time=datetime.utcnow(),
+            ...     end_time=datetime.utcnow() + timedelta(hours=1))
+
+            Creating a weekly exclusion:
+
+            >>> exclusion = tio.v3.vm.exclusions.create(
+            ...     'Example Weekly Exclusion',
+            ...     ['127.0.0.1'],
+            ...     frequency='weekly',
+            ...     weekdays=['mo', 'we', 'fr'],
+            ...     start_time=datetime.utcnow(),
+            ...     end_time=datetime.utcnow() + timedelta(hours=1))
+
+            Creating a monthly esxclusion:
+
+            >>> exclusion = tio.v3.vm.exclusions.create(
+            ...     'Example Monthly Exclusion',
+            ...     ['127.0.0.1'],
+            ...     frequency='monthly',
+            ...     day_of_month=1,
+            ...     start_time=datetime.utcnow(),
+            ...     end_time=datetime.utcnow() + timedelta(hours=1))
+
+            Creating a yearly exclusion:
+
+            >>> exclusion = tio.v3.vm.exclusions.create(
+            ...     'Example Yearly Exclusion',
+            ...     ['127.0.0.1'],
+            ...     frequency='yearly',
+            ...     start_time=datetime.utcnow(),
+            ...     end_time=datetime.utcnow() + timedelta(hours=1))
+        '''
+
+        # construct payload schedule based on enable
+        if enabled is True:
```

Comment:
if enabled. Similar in elif.

Reply:
enable is boolean parameter so using is the keyword we are also checking the type parameter.